app/build.gradle

```diff
@@ -1,28 +1,54 @@
 apply plugin: 'com.android.application'
+apply plugin: 'realm-android'
 
 android {
-    compileSdkVersion 28
-    defaultConfig {
-        applicationId "com.example.ayush.weatherapp"
-        minSdkVersion 25
-        targetSdkVersion 28
-        versionCode 1
-        versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
+  compileSdkVersion 28
+  defaultConfig {
+    applicationId "com.example.ayush.weatherapp"
```

Comment:
com.example is reserved packagename and is not suitable. change to com.ayush.weatherapp

Reply:
:ok: